Readme.md

```diff
@@ -38,6 +38,7 @@ This module lets you connect to web services using SOAP.  It also provides a ser
   - [ClientSSLSecurity](#clientsslsecurity)
   - [WSSecurity](#wssecurity)
   - [WSSecurityCert](#wssecuritycert)
+  - [NtlmSecurity](#ntlmsecurity)
```

Comment:
Please stay with the current naming scheme and rename `NtlmSecurity` to `NTLMSecurity` (as introduced by `ClientSSLSecurity` and `WSSecurity`)